src/pages/JournalEntryFormPage/UpdateJournalEntryForm/UpdateJournalEntryForm.tsx

```diff
@@ -50,16 +51,20 @@ const UpdateJournalEntryForm: React.FC<UpdateJournalEntryFormProps> = (props) =>
             garlandPose: garlandPose,
             prenatalVitamins: prenatalVitamins,
             probiotics: probiotics,
-        }
+        };
 
-        try {
-            updateJournalEntry.mutate(updatedJournalEntry)
-            handleSubmitResults("success")
-        } catch (error) {
-            console.log(error)
-            handleSubmitResults("error")
-        }
-    }
+        if (JSON.stringify(updatedJournalEntry) !== JSON.stringify(previousEntry)) {
```

Comment:
Easiest way at the current moment to check if two objects are equal. Ideally wouldn't be able to click submit until they change the data. This will work for now though.